### ISSUE TITLE
Translation issue with PageKit Blog extension

### ISSUE THREAD

**Issue by theofruitrouge** [biskuitorg/biskuit#87]
There is a problem with one string "Written on" from the original PageKit Blog extension.
Site is setup as French (France). Other strings are translated as expected in the blog interface (Read more, etc.) but not this one.
No clue why...

<img width="178" alt="image" src="https://user-images.githubusercontent.com/21720326/111037534-c85a9200-8424-11eb-8221-8aa874a55b64.png">

Biskuit : (pk) 1.0.19
Host : Apache 2.4.46
PHP : 7.4.16
Blog extension : 1.0.7
Using (to date) .htaccess from PageKit.

**Comment by tobbexiv** [comment]
The translations of the original pagekit are not up-to-date. This string seems to simply not being included in the language files. We need to check the translations and also set-up a proper translation environment.

**Comment by mirkobrombin** [comment]
We could request a free instance of WebLate as it is an Open source project.

**Comment by tobbexiv** [comment]
Alternatively Transifex as it was used before and tool support to fetch translations should already be there. It's also free for open source projects.

**Comment by mirkobrombin** [comment]
> Alternatively Transifex as it was used before and tool support to fetch translations should already be there. It's also free for open source projects.

I have never used it, do you have any experience?

**Comment by theofruitrouge** [comment]
The weird thing (why I created this issue) is that I run the same extension on PageKit and the string is translated correctly. So does it comes from a code modification in biskuit ?

<img width="115" alt="image" src="https://user-images.githubusercontent.com/21720326/111071769-fc42bf80-84d7-11eb-9077-7b2daf52c6f0.png">

PageKit : 1.0.18
Host : Apache
PHP : 7.4.16
Blog extension : 1.0.7

**Comment by theofruitrouge** [comment]
After some research, the theme UIkit 3 Cards by Sven Suchan (v 1.0.1) was the cause, because it rewrites the string to make the author disappear but does not provide translations. I will contact Sven regarding this.

**Comment by theofruitrouge** [comment]
PS : I will contribute to the translations if needed 👍

**Comment by mirkobrombin** [comment]
I reopen the issue only as a reminder for Transifex

**Comment by tobbexiv** [comment]
> I have never used it, do you have any experience?

Yes, I used it for my own porjects as well as participating in other projects. I could set it up if desired.

**Comment by mirkobrombin** [comment]
> > I have never used it, do you have any experience?
> 
> Yes, I used it for my own porjects as well as participating in other projects. I could set it up if desired.

It would be greatly appreciated

**Comment by tobbexiv** [comment]
Project will be at https://www.transifex.com/biskuit/biskuit/
The main translation file is https://www.transifex.com/biskuit/biskuit/biskuit-core/ and is set-up to automatically sync with https://github.com/biskuitorg/biskuit/blob/dev/app/system/languages/messages.pot. This means that new strings will be sent to transifex automatically once merged to dev. This also means that removal of string from this file might affect translations.


Next steps:
* Add languages and upload existing translations
* Set-up https://github.com/biskuitorg/translations to have the tools to fetch the translations from Transifex (and also have the translations available).
* Remove translations from core repo and update "build" command to fetch translations first.

Alternatively to having translations in separate repo: Add command to fetch translations from transifex directly to biskuit. Advantage: This logic could be re-used by extensions. Disdvantage: It is required to store the access token somewhere, so using this approach would be a multiple-step command line process.

Once done for the core project, repeat the steps for blog, ...

**Comment by mirkobrombin** [comment]
We could use GitHub Actions for the whole build process. Maybe provide one continous version at each commit and one per tag release.

Not specific to translations but we can use GitHub Actions for Biskuit release, with npm and composer packages already installed (like Pagekit did).

@francescomasala could take the job I think.
